link-grammar/linkage/linkage.c

```diff
@@ -692,12 +692,16 @@ static void compute_chosen_words(Sentence sent, Linkage linkage,
 	 * to facilitate using diff on sentence batch runs. */
 	if (test_enabled("removeZZZ"))
 	{
-		for (i=0; i<linkage->num_links; i++)
+		const char* ZZZ = linkgrammar_get_dict_define(sent->dict, EMPTY_CONNECTOR);
```

Comment:
Since this is just a test code, it can remain with `linkgrammar_get_dict_define()`.